examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,29 +14,40 @@
   },
```

Comment:
<p>dims and coords seem to have been ignored, could be a bug in ArviZ. I'll look into that. Do change this for the plot anyway though, even if at least for now the labels are wrong. Using <code>plot_posterior</code>  like we did above should work here too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>okayy</p>

Reply:
ArviZ ignores the dimensions for sample stats: https://github.com/arviz-devs/arviz/blob/main/arviz/data/io_pymc3.py#L323 it may be to avoid name collisions though.

Reply:
<p>do you think that could use some fixing, maybe an informative  warning or error could be given during a name collision. Because in real time models and stuff diagnostics running well, following general practices of arviz/pymc3 seems to be actually very important</p>

Reply:
Yes, we could fix that in ArviZ and have it use coords and dims also in the sample stats, either with a try except or checking things more carefully. But this is very low priority, compound steps are not that common and we have to make sure to implement that in a way that it never backfires.